src/test/regress/spec/isolation_enable_ddl_propagation.spec

```diff
@@ -0,0 +1,18 @@
+setup
+{
+	SELECT result from run_command_on_all_nodes('ALTER SYSTEM SET citus.enable_ddl_propagation TO ON');
+	SELECT result from run_command_on_all_nodes('SELECT pg_reload_conf()');
+}
+
+session "s1"
+step "s1-begin"
+{
+	BEGIN;
+}
+
+step "s1-commit"
+{
+	COMMIT;
+}
+
+permutation "s1-begin" "s1-commit"
```

Comment:
do we need those s1-begin s1-commit steps and the permutation?

Reply:
When I remove those lines, I get error

Reply:
A select1 somewhere would also work I guess, but the important thing is that there is at least one permutation, otherwise the setup stage never gets executed. Would be good to add a comment explaining that. Something like:

> The code we care about is in the `setup` stage, because it needs to be executed outside of a transaction. For the `setup` stage to be executed we need at least one permutation though. So we create a simple one without anything interesting.

Reply:
Fixed as @JelteF suggested